notebooks/405-paddleOCR-webcam/405-paddleOCR-webcam.ipynb

```diff
@@ -0,0 +1,557 @@
+    "        print(\"Downloading the pre-trained model... May take a while...\")\n",
+    "\n",
+    "        # create a directory\n",
+    "        if not base_model_dir.is_dir():\n",
+    "            os.makedirs(\"model\")\n",
+    "        urllib.request.urlretrieve(model_url, f\"model/{model_name} \")\n",
+    "        print(\"Model Downloaded\")\n",
+    "\n",
+    "        file = tarfile.open(f\"model/{model_name} \")\n",
+    "        res = file.extractall(\"model\")\n",
+    "        file.close()\n",
+    "        if (not res):\n",
+    "            print(f\"Model Extracted to {model_file_path}.\")\n",
+    "        else:\n",
+    "            print(\"Error Extracting the model. Please check the network.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e541150c-0f98-41c6-a97c-97acb26efd2f",
+   "metadata": {},
+   "source": [
+    "#### Download the Network for Text **Detection**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "02fe27ea-0aaf-4ecb-bce2-858d70c84e93",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Directory where model will be downloaded\n",
+    "\n",
+    "det_model_url = \"https://paddleocr.bj.bcebos.com/dygraph_v2.0/ch/ch_ppocr_mobile_v2.0_det_infer.tar\"\n",
+    "det_model_file_path = Path(\"model/ch_ppocr_mobile_v2.0_det_infer/inference.pdmodel\")\n",
+    "\n",
+    "run_model_download(det_model_url, det_model_file_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f9c5c83a-961c-4d98-8b20-5e96c8ef71f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# initialize inference engine for text detection\n",
+    "det_ie = Core()\n",
+    "det_net = det_ie.read_model(model=det_model_file_path)\n",
+    "det_compiled_model = det_ie.compile_model(model=det_net, device_name=\"CPU\")\n",
+    "\n",
+    "# get input and output nodes for text detection\n",
+    "det_input_layer = det_compiled_model.input(0)\n",
+    "det_output_layer = det_compiled_model.output(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ec5c940-626c-4cf7-a90f-833200969846",
+   "metadata": {},
+   "source": [
+    "#### Download the Network for Text **Recognition**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89c0a07a-8186-47b5-ad95-f104a84d13d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rec_model_url = \"https://paddleocr.bj.bcebos.com/dygraph_v2.0/ch/ch_ppocr_mobile_v2.0_rec_infer.tar\"\n",
+    "rec_model_file_path = Path(\"model/ch_ppocr_mobile_v2.0_rec_infer/inference.pdmodel\")\n",
+    "\n",
+    "run_model_download(rec_model_url, rec_model_file_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d196913-6542-4177-87ab-c5aa1994f8e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize the Paddle recognition inference on CPU\n",
+    "rec_ie = Core()\n",
```

Comment:
There is no need to initialize openvino many times. It can be initialized once (above)